src/main/java/org/jenkinsci/plugins/scriptsecurity/sandbox/groovy/SandboxInterceptor.java

```diff
@@ -114,6 +116,14 @@
                 throw StaticWhitelist.rejectStaticMethod(foundDgmMethod);
             }
 
+            // allow calling Closure elements of Maps as methods
+            if (receiver instanceof Map) {
+                Object element = onMethodCall(invoker, receiver, "get", method);
```

Comment:
Not technically correct to delegate to `onMethodCall` here: if the whitelist did not pass this particular `Map.get(Object)` call but the receiver _was_ allowed to work in some other way like the `invokeMethod` below, then we would be breaking something that worked before. OTOH `Map.get(Object)` at the interface level is whitelisted, so using the standard whitelist there is no way this would not pass.

Just another example of the problem noted in https://github.com/jenkinsci/groovy-sandbox/issues/7#issuecomment-39252544.